### DISCUSSION TITLE
Wiki - Compilation options - PXX2 protocol support

### DISCUSSION THREAD

**Discussion by 2xWhiskey** [EdgeTX/edgetx#2381] [Q&A] [answered]
I have a question about:

// PXX2 protocol support
PXX2:BOOL=OFF

That is in the Wiki at [Compilation-options](https://github.com/EdgeTX/edgetx/wiki/Compilation-options)

I do not know if it is current, but would it prevent me from using the PXX2 TX & RX UART connection on my radio's internal RF module?  I am still trouble shooting my DIY DupleTX module build for my Taranis X-Lite Pro.

Thank you.

**Comment by 2xWhiskey** [comment, marked as answer]
I've successfully been able to use the PXX2 lines.  So question answered.

**Reply by pfeerick** [reply]
So the option had no effect? I would have guessed it shouldn't, but there was some serial UART stuff in there so wasn't sure.

**Reply by 2xWhiskey** [reply]
I did not alter or tamper with anything EdgeTX-build related.  I was just trying to find and eliminate possibilities from a noob's perspective.
